thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/spec/PercentageChangeRuleAnomalyFilterSpec.java

```diff
@@ -21,9 +21,11 @@
 
 public class PercentageChangeRuleAnomalyFilterSpec extends AbstractSpec {
   private String timezone = "UTC";
-  private double threshold = Double.NaN;
   private String offset;
   private String pattern= "UP_OR_DOWN";
+  private double threshold = 0.0; // by default set threshold to 0 to pass all anomalies
+  private double upThreshold = Double.NaN;
```

Comment:
Why default value is NaN?

Reply:
Most of the filters are one side unless the threshold is overridden by upThreshold or downThreshold. If they are not NaN, then we know it's overridden.